tenant_users/permissions/models.py

```diff
@@ -18,59 +19,60 @@ class Meta:
     # permissions matching the current schema, which means that this
     # user has no authorization, so we catch this exception and return
     # the appropriate False or empty set
+    @cached_property
     def _get_tenant_perms(self):
```

Comment:
nit but a property should not have a `get` prefix imo

Reply:
Sure, thanks for pointing that out